### ISSUE TITLE
Add instructions on what to setup in Pinecone

### ISSUE THREAD

**Issue by Designer023** [pashpashpash/vault-ai#15]
I've not used Pinecone before so I don't know the right config and I keep getting issues. Can you add a guide so that a beginner can create a pinecone index please 👍

**Comment by pashpashpash** [comment]
@Designer023 Good call, I added a line in the readme about this. When setting up your pinecone index, use a vector size of 1536 and keep all the default settings the same.
